tslint.json

```diff
@@ -1,9 +1,42 @@
 {
     "defaultSeverity": "error",
     "extends": [
-        "tslint:recommended"
+      "tslint:recommended",
+      "tslint-config-airbnb",
+      "tslint-react"
     ],
+    "linterOptions": {
+      "exclude": [
+        "node_modules/**",
+        "typings/**",
+        "App/Sagas/**/*",
+        "App/Redux/**/*",
+        "App/Navigation/**/*",
+        "App/Models/**/*",
+        "App/SB/**/*",
+        "**/*.json"
```

Comment:
Okay, kind of a weak first go here. But i wanted to avoid having to basically PR a file in pretty much every folder of the repo on the first go. It also should save us some headaches merging with the Comments PR still outstanding. We can add these in later.